Feat/upgrade molecule to v5

bngameni opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->

## Description
<!--- Describe your changes in detail -->
Upgrade molecule version to v5

## Motivation and Context
<!--- Why is this change required? What problem does it solve? -->
<!--- If it fixes an open issue, please link to the issue here. -->
Use latest molecule available for perform testing

## How Has This Been Tested?
<!--- Please describe in detail how you tested your changes. -->
<!--- Include details of your testing environment, the tests you ran to -->
<!--- see how your change affects other areas of the code, etc. -->
In CI workflow

## Screenshots (if appropriate):

## Types of Changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Tests

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

Commit:
fix: add changed_when for fix ansible lint error

Commit:
fix: remove flat8 option for recent molecule version

Commit:
tests: add debian 12 in workflow

Commit:
docs: update testing badge in readme

Commit:
review: upgrade default version to latest

Commit:
fix: remove too old images in testing workflow

Commit:
fix: add php role in requirements

Commit:
fix: change dockerfile for allow run's in ubuntu

Commit:
fix: unable to create container for centos8

Commit:
feat: add debian 10, ubuntu 20,18 in testing workflow

Commit:
feat: upgrade to molecule 5

ghost commented:
Hello,

Thank you for creating the Pull Request.

Some the commits in the Pull Request have already been integrated in a previous Pull Request.

To have a cleaner PR, could you please create a Pull Request with only the new changes since the last integrated Pull Request ?

Thank you,
Regards,

bngameni commented:
> Hello,
> 
> Thank you for creating the Pull Request.
> 
> Some the commits in the Pull Request have already been integrated in a previous Pull Request.
> 
> To have a cleaner PR, could you please create a Pull Request with only the new changes since the last integrated Pull Request ?
> 
> Thank you, Regards,

Hello,
Done for new cleaner PR
